server/commands/admin.py

```diff
@@ -414,3 +416,47 @@ def ooc_cmd_baninfo(client, arg):
         else:
             msg += 'Unban date: N/A'
         client.send_ooc(msg)
+
+def ooc_cmd_gimp(client, arg):
+    if not client.is_mod:
+        raise ClientError('You must be authorized to do that.')
+    elif len(arg) == 0:
+        raise ArgumentError('You must specify a target.')
+    try:
+        if len(arg) == 12:
+            targets = client.server.client_manager.get_targets(client, TargetType.IPID, arg, False)
+        elif len(arg) < 12 and arg.isdigit():
+            targets = client.server.client_manager.get_targets(client, TargetType.ID, int(arg), False)
+        else:
+            raise ArgumentError()
+    except:
```

Comment:
I don't like this catch-all. A legitimate error in `get_targets` would then be hidden behind "you typed it wrong."
```suggestion
    except ArgumentError:
```